app/main.py

```diff
@@ -44,6 +44,7 @@ def resize(self) -> None:
         for node in table:
             if node is not None:
                 self.__setitem__(node.key, node.value)
+                node.hash = self.get_hash(node.key)
```

Comment:
you didn't fix the problem: your `get_hash` function returns index and not hash. You just need to use the `hash` function